dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -35,17 +35,17 @@ warning.could.not.thin.application=CWWKC0251W: Could not convert the application
 warning.could.not.thin.application.explanation=The application could not be converted to a thin application. The application will continue to run from the original JAR. 
 warning.could.not.thin.application.useraction=Attempt to resolve the issue in the exception message and either restart the server or update the application. 
 
-error.need.springboot.version.15=CWWKC0252E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-1.5 to be configured.
-error.need.springboot.version.15.explanation=The application requires Spring Boot version 1.5, but a wrong version of the Spring Boot feature is configured in the server.xml.  
-error.need.springboot.version.15.useraction=The springBoot-1.5 feature needs to be configured in the server.xml.
+error.need.springboot.version.15=CWWKC0252E: The application failed to start because the wrong version of the Spring Boot feature is configured in the server.xml file. The application requires the feature springBoot-1.5 to be configured.
+error.need.springboot.version.15.explanation=The application requires Spring Boot version 1.5, but a wrong version of the Spring Boot feature is configured in the server.xml file.  
+error.need.springboot.version.15.useraction=Configure the springBoot-1.5 feature in the server.xml file.
```

Comment:
The comments above also apply to lines 42-44.

Reply:
To clarify that the user should also remove the incorrect features, add, "Remove any extra Spring Boot features." or "Remove any other Spring Boot features."